src/Sulu/Bundle/MediaBundle/Tests/Functional/Media/Storage/S3StorageTest.php

```diff
@@ -19,14 +19,15 @@
 
 class S3StorageTest extends SuluTestCase
 {
-    /**
-     * @var string
-     */
-    protected static $class = S3Kernel::class;
```

Comment:
Symfony 7 has this typehinted with `string` but to support Symfony 5 and 6 we can not just add a typehint as that is not allowed to change a variable from `mixed` -> `string`. So we need to change how we set the kernel class and how we reset it.